Makefile

```diff
@@ -212,7 +212,7 @@ docs/spec: go.mod
 # Beats synchronisation.
 ##############################################################################
 
-BEATS_VERSION?=8.10
+BEATS_VERSION?=8.10.0
```

Comment:
Is this right? Pretty sure this is meant to be a branch name

Reply:
Hmm, you are correct. Fixed in https://github.com/elastic/apm-server/pull/11418